### PR TITLE
fix: Fix typing error of generators

### PR DESCRIPTION
**Pull request by ywwhack**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

#### Behavior Before Change
`workspaceToCode` method  is not inferred automatic

image

image

#### Behavior After Change
`workspaceToCode` method  is inferred automatic

image

### Reason for Changes

Fix wrong ts definition for exported generator.

### CODE REVIEW

**Comment by ywwhack** [comment]
@googlebot I signed it!

File: typings/blockly.d.ts [hunk 1]
```diff
@@ -6635,6 +6635,31 @@ declare module Blockly.Generator {
      * Category to separate generated function names from variables and procedures.
      */
     var NAME_TYPE: any /*missing*/;
+
+    /**
+     * instance of javascript generator
+     */
+     var javascript: Blockly.Generator;
```
Comment (samelhusseini) [inline typings/blockly.d.ts:6642]: Can you move these instances into the generator typings file itself?

That way if one only includes some generators but not others, they don't get them all

File: typings/dart.d.ts [hunk 2]
```diff
@@ -12,4 +12,4 @@
 /// <reference path="core.d.ts" />
 
 import * as Blockly from './core';
-export = Blockly.Generator;
+export = Blockly.Generator.dart;
```
Comment (samelhusseini) [inline typings/dart.d.ts:15]: instead: 
```
const dart: Blockly.Generator;
export = dart;
```

File: typings/blockly.d.ts (hunk 1 above)
Comment (ywwhack) [reply typings/blockly.d.ts]: Done

**Review by samelhusseini — APPROVED** [review]

**Comment by samelhusseini** [comment]
Thank you for the PR!

**Pull request merged**